### PR TITLE
Update the SARS-CoV-2 variant discovery tutorial

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Comment by wm75** [comment]
@hexylena I'm trying to link to an anchor element inside the faqs, which also has a space in its name:

with a literal space I'm getting:
https://github.com/galaxyproject/training-material/actions/runs/4959278452/jobs/8873169145?pr=4092#step:9:29

but when I encode it as %20, I'm getting a jekyll parsing issue, like here:
https://github.com/galaxyproject/training-material/actions/runs/4958699204/jobs/8871891089#step:9:35

Do you know what kind of syntax would make linting succeed?

**Comment by wm75** [comment]
Puh, done with this except for the failing linting.

File: topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md [hunk 1]
```diff
@@ -80,789 +130,895 @@ Any analysis should get its own Galaxy history. So let's start by creating a new
...
+>   - from a shared data library on the Galaxy server you are working on
+>
+>   In all of these cases you will also have to organize the imported data into a dataset collection like explained in detail for the suggested example data.
+>
+>   > <details-title>Data logistics</details-title>
+>   >
+>   > A detailed explanation of all of the above-mentioned options for getting your data into Galaxy and organizing it in your history is beyond the scope of this tutorial.
+>   > If you are struggling with getting your own data set up like shown for the example data in this section, please:
+>   > - Option 1: Browse some of the material on [Using Galaxy and Managing your Data]({% link topics/galaxy-interface %})
+>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy#data%20upload %}) and on [collections]({% link faqs/galaxy#collections %})
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md:149]:
```suggestion
>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy.md %}#data%20upload) and on [collections]({% link faqs/galaxy.md %}#collections)
```

File: topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md [hunk 2]
```diff
@@ -80,789 +130,895 @@ Any analysis should get its own Galaxy history. So let's start by creating a new
...
->
-> More details about the workflows, including benchmarking of the tools, can be found on [covid19.galaxyproject.org](https://covid19.galaxyproject.org/genomics/global_platform/#methods)
 {: .details}
 
-> <hands-on-title>From FASTQ to annotated AVs</hands-on-title>
+{% include _includes/cyoa-choices.html option1="tiled-amplicon Illumina paired-end" option2="tiled-amplicon ONT" option3="WGS Illumina paired-end" option4="WGS Illumina single-end" default="tiledamplicon-Illumina-pairedend" text="Now that you have imported the data and the corresponding workflow of your choice, please select the type of your input data so that we can adjust a few parts of this tutorial that are dependent on the nature of your data:" %}
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md:436]: wow! this is huge

Comment (shiltemann) [reply topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md:436]: love it :)

File: topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md [hunk 3]
```diff
@@ -80,789 +130,895 @@ Any analysis should get its own Galaxy history. So let's start by creating a new
...
+>   - from a shared data library on the Galaxy server you are working on
+>
+>   In all of these cases you will also have to organize the imported data into a dataset collection like explained in detail for the suggested example data.
+>
+>   > <details-title>Data logistics</details-title>
+>   >
+>   > A detailed explanation of all of the above-mentioned options for getting your data into Galaxy and organizing it in your history is beyond the scope of this tutorial.
+>   > If you are struggling with getting your own data set up like shown for the example data in this section, please:
+>   > - Option 1: Browse some of the material on [Using Galaxy and Managing your Data]({% link topics/galaxy-interface %})
+>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy.md %}#data%20upload) and on [collections]({% link faqs/galaxy.md %}#collections)
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md:149]:
```suggestion
>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy.md %}#data%20upload) and on [collections]({% link faqs/galaxy/index.md %}#collections)
```

File: topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md [hunk 4]
```diff
@@ -80,789 +130,895 @@ Any analysis should get its own Galaxy history. So let's start by creating a new
...
+>   - from a shared data library on the Galaxy server you are working on
+>
+>   In all of these cases you will also have to organize the imported data into a dataset collection like explained in detail for the suggested example data.
+>
+>   > <details-title>Data logistics</details-title>
+>   >
+>   > A detailed explanation of all of the above-mentioned options for getting your data into Galaxy and organizing it in your history is beyond the scope of this tutorial.
+>   > If you are struggling with getting your own data set up like shown for the example data in this section, please:
+>   > - Option 1: Browse some of the material on [Using Galaxy and Managing your Data]({% link topics/galaxy-interface %})
+>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy.md %}#data%20upload) and on [collections]({% link faqs/galaxy/index.md %}#collections)
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md:149]:
```suggestion
>   > - Option 2: Consult the FAQs on [uploading data]({% link faqs/galaxy/index.md %}#data%20upload) and on [collections]({% link faqs/galaxy/index.md %}#collections)
```

**Comment by shiltemann** [comment]
> @hexylena I'm trying to link to an anchor element inside the faqs, which also has a space in its name:
> 
> with a literal space I'm getting: https://github.com/galaxyproject/training-material/actions/runs/4959278452/jobs/8873169145?pr=4092#step:9:29
> 
> but when I encode it as %20, I'm getting a jekyll parsing issue, like here: https://github.com/galaxyproject/training-material/actions/runs/4958699204/jobs/8871891089#step:9:35
> 
> Do you know what kind of syntax would make linting succeed?

looks like you got it to work now, but we should really slugify those to avoid spaces (TODO for myself)

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Whoo! Thanks a lot @wm75!! :tada:

**Pull request merged**